doc/design/restore_existing.md

```diff
@@ -0,0 +1,17 @@
+Restore From Existing Backup
+======
+
+We want to restore an etcd cluster from existing backup that operator has saved before.
+
+We add a field in BackupPolicy:
+```Go
+type BackupPolicy struct {
+    // InitPV...
+    InitPV string `json:"initPV"`
```

Comment:
So there are two cases I can think of.

1. I created a cluster named `etcd-A`. The cluster is still running. I want to have a clone of that cluster for testing/debugging purpose. So I want to create `etcd-B` from the latest `etcd-A` backup.

2. I created `etcd-A`. I accidentally removed `etcd-A`. Now I want to recreate a cluster named `etcd-A` (or `etcd-A1`) from the latest backup.

For both of the two cases, I only know that I created the cluster by submitting a TPR named `etcd-A` with backup enabled and nothing else.